### PR TITLE
fix(badges): update themeing of badges

### PR DESCRIPTION
**Pull request by SamJacobs**
@dzucconi You might hate this

### CODE REVIEW

File: src/components/Badge/Badge.tsx [hunk 1]
```diff
@@ -3,17 +3,21 @@ import classNames from 'classnames';
...
 
+export const TRENDING_THEME = 'trending';
+export const BESTSELLER_THEME = 'bestseller';
+
 const THEMES = {
-  bestseller: 'Bestseller',
-  trending: 'Trending',
+  [BESTSELLER_THEME]: 'Bestseller',
+  [TRENDING_THEME]: 'Trending',
   'fall-essential': 'Fall Essential'
```
Comment (dzucconi) [inline src/components/Badge/Badge.tsx]: why not also `fall-essential`?

Comment (SamJacobs) [reply src/components/Badge/Badge.tsx]: because i didnt need it elsewhere, but sure I can do it

Comment (dzucconi) [reply src/components/Badge/Badge.tsx]: LGTM otherwise

**Pull request merged**